overview/glossary.md

```diff
@@ -0,0 +1,42 @@
+| Attestation                                                                           | A verification of something as true or authentic by evidence.  In uPort attestations cryptographically signed claims.                                                                                     |
+| Application Identity                                                                  | Just like a user has a uPort identity, applications and dApps that interact with uPort users need their own identity a.k.a (key/pair)                                                                     |
+| Claim                                                                                 |  A claim is a record of an attestation that has been verified.                                                                                                                                            |
+| Credentials                                                                           | Credentials are pieces of information about a uPort user that may be requested and/or verified by applications.  They are, in their essence, signed claims that are stored with the uPort user's profile. |
+| dAPP                                                                                  | Distributed Application, generally one that relies on serverless architectures.                                                                                                                           |
+| DID                                                                                   | Decentralized Identifier used to retrieve a DID document as defined by the [W3C Community Group](https://w3c-ccg.github.io/did-spec/)                                                                     |
+| DID Document                                                                          | A DID document is an aggregation of specific types of interactions with a decentralized identifier (DID).  To see one implementation of how uPort resolves a DID document, see our [Ethr-DID-Resolver](https://github.com/uport-project/ethr-did-resolver ) library.  Other types of DID resolvers offered by uPort: [uPort-DID-Resolver](https://github.com/uport-project/uport-did-resolver),  [muPort-DID-Resolver](https://github.com/uport-project/muport-did-resolver), [Secp256k1-DID-Resolver](https://github.com/uport-project/secp256k1-did-resolver) |
+| [DID-Resolver](https://github.com/uport-project/did-resolver)                         | A library that provides a simple interface to resolve DID documents from a decentralized identifier.                                                                                                      |
+| [Eth-DID-Resolver](https://github.com/uport-project/eth-did-resolver)                 | A library that takes an ethereum address as a decentralized identifier and wraps it in a DID document.                                                                                                       |
```

Comment:
I think we should probably get rid of this library. So remove the EthDID Resolver to avoid confusion